src/app/(dashboard)/layout.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode} from "react";
+import MyGetServerSession from "@/lib/myGetServerSession";
+import {notFound} from "next/navigation";
+import Link from "next/link";
+import {Icons, Icon} from "@/components/Icons";
+import NavbarListItem, {navbarListItemProps} from "@/app/(dashboard)/navbarlistitem";
+
+interface LayoutProps {
+    children: ReactNode
+}
+
+const options:navbarListItemProps[] = [{
+    id: 1,
+    name:'Add a Friend',
+    Icon: 'UserPlus',
+    href: '/dashboard/add',
+},
+    {
+        id: 1,
+        name:'Add a Friend',
+        Icon: 'Logo',
+        href: '/dashboard/add',
+    }
+]
+
+const Layout = async ({children}: LayoutProps)=>{
+    const session = await MyGetServerSession();
+    if (!session){
+        notFound();
+    }
+    return <div className='w-full flex h-screen'>
+        <div className='flex h-full w-full max-w-xs frow flex-col gap-y-5 overflow-y-auto border-r border-gray-200 bg-white px-6'>
```

Comment:
encapsulate the markup in a separate file in the same dir - all this styling is filthy and defeats the purpose of separating HTML from CSS in the first place